apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/codegen/kotlin/adapter/AdapterCommon.kt

```diff
@@ -107,7 +107,7 @@ internal fun readFromResponseCodeBlock(
                 context.layout.variableName(property.info.responseName),
                 context.resolver.resolveIrType(property.info.type).copy(nullable = !property.info.type.isOptional()),
             )
-            beginControlFlow("if·(%L.%M(emptySet(),·$__typename))", property.condition.codeBlock(), evaluate)
+            beginControlFlow("if·(%L.%M($customScalarAdapters.variables(),·$__typename))", property.condition.codeBlock(), evaluate)
```

Comment:
How come this doesn't impact a bunch of `.exepected` files?

Reply:
Ah I guess it does actually 😅

Reply:
Yup, my bad, forgot to regenerate those, doing it now.